sdcm/cluster.py

```diff
@@ -4329,6 +4329,9 @@ def node_setup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600)
 
         node.update_repo_cache()
 
+        if node.distro.is_ubuntu:
```

Comment:
Maybe we should install it as part of the nemesis that needs it ?

Reply:
OK, let's do it this way.